lib/src/screens/cache_store/app_cache_store_screen.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:flutter/material.dart';
+import 'package:mustang_core/mustang_widgets.dart';
+import 'package:mustang_viewer/src/screens/cache_store/app_cache_store_service.dart';
+import 'package:mustang_viewer/src/screens/shared_services/next_search_index_action.dart';
+import 'package:mustang_viewer/src/screens/shared_services/previous_search_index_action.dart';
+import 'package:mustang_viewer/src/shared_widgets/app_progress_indicator.dart';
+import 'package:mustang_viewer/src/shared_widgets/model_view.dart';
+import 'package:mustang_viewer/src/utils/app_constants.dart';
+
+import 'app_cache_store_state.state.dart';
+
+class AppCacheStoreScreen extends StatefulWidget {
+  const AppCacheStoreScreen({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  State<AppCacheStoreScreen> createState() => _AppCacheStoreScreenState();
+}
+
+class _AppCacheStoreScreenState extends State<AppCacheStoreScreen> {
+  final ScrollController dataViewScrollController = ScrollController();
+  final TextEditingController dataViewSearchTextController =
+      TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    return StateProvider<AppCacheStoreState>(
+      state: AppCacheStoreState(),
+      child: Builder(
+        builder: (BuildContext context) {
+          AppCacheStoreState? state =
+              StateConsumer<AppCacheStoreState>().of(context);
+
+          if (state!.appCacheStore.busy) {
+            const Scaffold(
+              body: AppProgressIndicator(),
+            );
+          }
+
+          if (state.appCacheStore.errorMsg.isNotEmpty) {
+            Text(state.appCacheStore.errorMsg);
+          }
+
+          return _body(state, context);
+        },
+      ),
+    );
+  }
+
+  Widget _body(AppCacheStoreState state, BuildContext context) {
+    String storeData = state.appCacheStore.data;
+    return Scaffold(
+      appBar: AppBar(),
+      body: SingleChildScrollView(
+        child: Column(
+          children: [
+            if (state.appCacheStore.data == '{}')
+              showInputDialog(context, state),
+            if (storeData != '{}')
+              Expanded(
+                child: ModelView(
+                  storeData,
```

Comment:
I think you have not rebased master changes. ModelView widget does not accept String anymore for data. 

`const ModelView(
    this.activeEvent,
    this.searchText,
    this.onSearchTextChange,
    this.scrollController,
    this.highlightIndices,
    this.indexOfSelectedHighlight,
    this.updateSelectedIndex,
    this.nextSearchIndexAction,
    this.previousSearchIndexAction,
    this.searchTextController, {
    Key? key,
  }) : super(key: key);

  final AppEvent activeEvent;
  final String searchText;
  final void Function(String term) onSearchTextChange;
  final ScrollController scrollController;
  final List<int> highlightIndices;
  final int indexOfSelectedHighlight;
  final void Function(int index) updateSelectedIndex;
  final NextSearchIndexAction nextSearchIndexAction;
  final PreviousSearchIndexAction previousSearchIndexAction;
  final TextEditingController searchTextController;`